SpatialGDK/Source/SpatialGDK/Public/SpatialView/ComponentData.h

```diff
@@ -24,7 +24,7 @@ struct ComponentDataDeleter
 using OwningComponentDataPtr = TUniquePtr<Schema_ComponentData, ComponentDataDeleter>;
 
 // An RAII wrapper for component data.
-class ComponentData
+class SPATIALGDK_API ComponentData
```

Comment:
Why do we need this now?

Reply:
It's the test, it looks through components on an entity and needs to do `GetComponentId` and `GetUnderlying` which aren't exported.